tests/core/config/startupFocus.js

```diff
@@ -0,0 +1,80 @@
+/* bender-tags: editor */
+// jscs:disable maximumLineLength
+/* bender-ckeditor-plugins: about,basicstyles,bidi,blockquote,clipboard,colorbutton,colordialog,div,enterkey,entities,find,flash,font,format,forms,horizontalrule,image,iframe,indent,justify,link,list,newpage,pagebreak,pastefromword,pastetext,removeformat,resize,toolbar,save,selectall,showblocks,smiley,sourcearea,specialchar,stylescombo,table,templates,undo,wysiwygarea */
+// jscs:enable maximumLineLength
+
+
+( function() {
+	'use strict';
+
+	function botInitializer( setup ) {
+		bender.editorBot.create( {
+			creator: 'replace',
+			name: setup.name,
+			config: Object.assign({
+				plugins: bender.plugins.join( ',' ),
+				startupFocus: setup.startupFocus
+			}, setup.extraConfig )
+		}, function( bot ) {
+			resume( function() {
+				if ( setup.startupFocus ) {
+					assert.isTrue( bot.editor.focusManager.hasFocus, 'editor is focused');
+				} else {
+					assert.isFalse( bot.editor.focusManager.hasFocus, 'editor is not focused' );
+				}
+				if ( focus === 'end' ) {
```

Comment:
This basically checks `focus` variable which is not defined anywhere, so the assert is not executed in any test (probably you mean `setup.startupFocus == 'end'`).